HerPublicWebsite.UnitTests/BusinessLogic/Services/DateHelperTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using HerPublicWebsite.BusinessLogic.Services.RegularJobs;
+using NUnit.Framework;
+
+namespace Tests.BusinessLogic.Services;
+
+public class DateHelperTests
+{
+    [Test]
+    public void StartOfMonthService_WhenCalledWithoutParameter_ReturnsAPreviousDate()
+    {
+        // arrange
```

Comment:
First letter should be a capital: `// Arrange`, `// Act`, `// Assert`

Reply:
In general, you should make sure to always check other similar code for established conventions.